teuthology/openstack/__init__.py

```diff
@@ -72,14 +76,19 @@ def enforce_json_dictionary(something):
             ' you are encouraged to add a comment if you want it to be'
             ' fixed.')
 
+def create_connection():
+    if openstack_connection is None:
+        raise RuntimeError("Did not find required openstack dependencies!")
```

Comment:
Maybe it's better to give a user more ideas which dependencies are needed, like tell about `openstacksdk` or give a hint that teuthology needs to be installed with `[openstack]` option. And no need to scary with bang symbol :-)

Reply:
Good idea - updated